scripts/bert/bertpass_gpu.cc

```diff
@@ -0,0 +1,450 @@
+/*
```

Comment:
Would you mind also updating index.rst with the usage for deploy.py?

Reply:
just updated, but please let me know if you think we can improve it.
I am not very sure how the custom graph pass should be shared (source code / library / in another place)

Reply:
How is the compilation of the custom graph pass currently triggered? Do you think it should be distributed as part of gluonnlp (or as now, "only" as part of the scripts). If so, ideally, pip should trigger compilation of the graph pass when installing gluonnlp. You could set this up via the setup.py. Maybe there are some missing pieces in the mxnet pip package preventing this to work. 

If you like to keep it in the scripts, you can also add a setup.py in the scripts folder.

It would be good to try this out.

Reply:
Thanks for the ideas. In the current version is not triggered, so yes, we need to provide a mechanism for that.
I think a setup.py would make more sense since this is specific for BERT-gpu and optional, and probably we do not want to affect the installation process.